### PR TITLE
SPA QS backlinks to Dev Center

### PR DESCRIPTION
**Pull request by avanscoy**
First edits for DR-1692

[DR-1692]: https://auth0team.atlassian.net/browse/DR-1692?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by avanscoy** [comment]
This PR needs reviewed by @dan-auth0

File: articles/quickstart/spa/vuejs/01-login.md [hunk 1]
```diff
@@ -20,7 +20,10 @@ useCase: quickstart
 ::: warning
 This quickstart is designed for using [Auth0 Vue](https://github.com/auth0/auth0-vue) with Vue 3 applications. If you are using Vue 2, please check out the [Vue 2 Tutorial with Auth0 SPA SDK](https://github.com/auth0/auth0-vue/blob/main/tutorial/vue2-login.md) instead.
 :::
+:::note
+For a deep dive into implementing user authentication in Vue, visit the [Vue.js Authentication By Example](https://developer.auth0.com/resources/guides/spa/vue/basic-authentication#integrate-vue-js-with-an-api-server/). This guide provides you with additional details, such as creating a signup button, protecting routes using different strategies, and using class components. 
+:::
```
Comment (frederikprijck) [inline articles/quickstart/spa/vuejs/01-login.md:25]: Should this point to a different link, and not the one to integrate it with an API? I would think the link without the anchor to be in line with the others. Also, what do you mean with class components in the context of Vue ?

File: articles/quickstart/spa/angular/interactive.md [hunk 2]
```diff
@@ -11,6 +11,10 @@ github:
   path: Sample-01
 ---
 
+:::note
+For a deep dive into implementing user authentication in Angular, visit the [Angular Authentication By Example](https://developer.auth0.com/resources/guides/spa/angular/basic-authentication). This guide provides information for additional tasks, like creating a signup button.
+:::
```
Comment (frederikprijck) [inline articles/quickstart/spa/angular/interactive.md:16]: Should we add this in the non-interactive version as well?

File: articles/quickstart/spa/vuejs/interactive.md [hunk 3]
```diff
@@ -20,6 +20,10 @@ files:
   - files/profile
 ---
 
+:::note
+For a deep dive into implementing user authentication in Vue, visit the [Vue.js Authentication By Example](https://developer.auth0.com/resources/guides/spa/vue/basic-authentication#integrate-vue-js-with-an-api-server/). This guide provides you with additional details, such as creating a signup button, protecting routes using different strategies, and using class components. 
+:::
```
Comment (frederikprijck) [inline articles/quickstart/spa/vuejs/interactive.md:25]: Should we drop the anchor here?

File: articles/quickstart/spa/react/interactive.md [hunk 4]
```diff
@@ -11,6 +11,9 @@ github:
   path: Sample-01
 ---
 
+:::note
+For a deep dive into implementing user authentication in React, visit the [React Authentication By Example](https://developer.auth0.com/resources/guides/spa/react/basic-authentication/). This guide provides you with additional details, such as creating a signup button, protecting routes using different strategies, and using class components.
```
Comment (frederikprijck) [inline articles/quickstart/spa/react/interactive.md]: Is it still correct that it shows using class components? I can't seem to find something that calls out class components, but I might not be familiar enough with developer.auth0.com.

**Review by frederikprijck — CHANGES_REQUESTED** [review]
Awesome seeing these links getting updated to point to developer.auth0.com! I added a couple of small comments, but this change is great as I think pointing people to `developer.auth0.com` as a next step is definetly a good idea!

File: articles/quickstart/spa/vuejs/01-login.md (hunk 1 above)
Comment (avanscoy) [reply articles/quickstart/spa/vuejs/01-login.md:25]: This one is my bad ^ thanks for catching it!

File: articles/quickstart/spa/angular/interactive.md (hunk 2 above)
Comment (avanscoy) [reply articles/quickstart/spa/angular/interactive.md:16]: Yes, I'm still wondering where the best place to add it would be. Dan requested the notes be moved to the top of the page. A good place is before content starts or right after the intro before the TOC.

File: articles/quickstart/spa/vuejs/interactive.md (hunk 3 above)
Comment (avanscoy) [reply articles/quickstart/spa/vuejs/interactive.md:25]: ^ this is a good question. I feel like the note would work here or before the QS requirements. What do you think?

File: articles/quickstart/spa/react/interactive.md (hunk 4 above)
Comment (avanscoy) [reply articles/quickstart/spa/react/interactive.md]: @dan-auth0 ^

Comment (DanOnCall) [reply articles/quickstart/spa/react/interactive.md]: 🤔 Hello, Fred! How do you mean using class components? That guide uses function components and React hooks.

File: articles/quickstart/spa/vuejs/interactive.md (hunk 3 above)
Comment (frederikprijck) [reply articles/quickstart/spa/vuejs/interactive.md:25]: I am not sure what you mean, the note is about adding user authentication, yet then links to `#integrate-vue-js-with-an-api-server/`, that isn't about user authentication. Is there any reason not to have it the same as with react for example, and have it point to https://developer.auth0.com/resources/guides/spa/vue/basic-authentication ?

File: articles/quickstart/spa/react/interactive.md (hunk 4 above)
Comment (frederikprijck) [reply articles/quickstart/spa/react/interactive.md]: @dan-auth0 But the comment mentions explicitly it's explaining class components, while it's not (anymore?).

> This guide provides you with additional details, such as creating a signup button, protecting routes using different strategies, **and using class components.**

Comment (frederikprijck) [reply articles/quickstart/spa/react/interactive.md]: Looking into it, it seems like this has always been incorrect, or the guide has been update. But let's take the opportunity to rectify this and ensure the paragraphs use the correct wording.

File: articles/quickstart/spa/vuejs/02-calling-an-api.md [hunk 5]
```diff
@@ -18,6 +18,10 @@ useCase: quickstart
 
 <!-- markdownlint-disable MD002 MD041 -->
 
+:::note
+For a deep dive into implementing user authentication in Vue, visit the [Vue.js Authentication By Example: Call an API.](https://developer.auth0.com/resources/guides/spa/vue/basic-authentication#integrate-vue-js-with-an-api-server/). This guide provides you with additional details, such as creating a signup button, protecting routes using different strategies, and using class components. 
+:::
```
Comment (frederikprijck) [inline articles/quickstart/spa/vuejs/02-calling-an-api.md]: Should this mention something like we do in React? People reading this quickstart are not looking into user authentication, but calling an API:

```suggestion
:::note
For a deep dive into making secure calls to an API from Vue, visit the [integrating Vue.js with an API server guide](https://developer.auth0.com/resources/guides/spa/react/basic-authentication#call-a-protected-api-from-react/). This guide provides you with additional details, such setting up a sample Express API server and getting test access tokens from the Auth0 Dashboard.
:::
```

**Review by frederikprijck — CHANGES_REQUESTED** [review]

File: articles/quickstart/spa/vuejs/interactive.md (hunk 3 above)
Comment (avanscoy) [reply articles/quickstart/spa/vuejs/interactive.md:25]: Apologies, I misunderstood. I thought you meant at this place in the article.

File: articles/quickstart/spa/react/interactive.md (hunk 4 above)
Comment (DanOnCall) [reply articles/quickstart/spa/react/interactive.md]: I got it now! The deprecated guide had a link to a repo that used class components. Yes, we can remove that part 👍 Thanks, Fred!

> This guide provides you with additional details, such as creating a signup button and protecting routes using React Router.

File: articles/quickstart/spa/angular/01-login.md [hunk 6]
```diff
@@ -158,6 +158,3 @@ The `user$` observable contains sensitive information and artifacts related to t
 Verify that you can display the `user.name` or [any other `user` property](/users/references/user-profile-structure#user-profile-attributes) within a component correctly after you have logged in.
 :::
 
-:::note
```
Comment (DanOnCall) [inline articles/quickstart/spa/angular/01-login.md:161]: @avanscoy The note is removed here but not added again in the file I think.

File: articles/quickstart/spa/angular/02-calling-an-api.md [hunk 7]
```diff
@@ -158,6 +158,3 @@ Your application will show an empty JSON object if you have not set any `user_me
 
 Please refer to the the [Auth0 API quickstarts](https://auth0.com/docs/quickstart/backend) to learn how to integrate Auth0 with your backend platform.
 
-:::note
```
Comment (DanOnCall) [inline articles/quickstart/spa/angular/02-calling-an-api.md:161]: Same situation here where the note is removed here but not added again.

File: articles/quickstart/spa/react/01-login.md [hunk 8]
```diff
@@ -14,6 +14,10 @@ useCase: quickstart
 ---
 <!-- markdownlint-disable MD002 MD034 MD041 -->
 
+:::note
+For a deep dive into implementing user authentication in React, visit the [React Authentication By Example](https://developer.auth0.com/resources/guides/spa/react/basic-authentication/). This guide provides you with additional details, such as creating a signup button, protecting routes using different strategies, and using class components.
```
Comment (DanOnCall) [inline articles/quickstart/spa/react/01-login.md]:
```suggestion
Visit the [React Authentication By Example](https://developer.auth0.com/resources/guides/spa/react/basic-authentication/) guide for a deep dive into implementing user authentication in React. This guide provides additional details on how to create a sign-up button and add route guards using React Router.
```

File: articles/quickstart/spa/angular/interactive.md [hunk 9]
```diff
@@ -11,6 +11,10 @@ github:
   path: Sample-01
 ---
 
+:::note
+For a deep dive into implementing user authentication in Angular, visit the [Angular Authentication By Example](https://developer.auth0.com/resources/guides/spa/angular/basic-authentication). This guide provides information for additional tasks, like creating a signup button.
```
Comment (DanOnCall) [inline articles/quickstart/spa/angular/interactive.md]:
```suggestion
Visit the [Angular Authentication By Example](https://developer.auth0.com/resources/guides/spa/angular/basic-authentication) guide for a deep dive into implementing user authentication in Angular. This guide provides additional details on how to create a sign-up button, add route guards, and call a protected API from Angular.
```

File: articles/quickstart/spa/react/interactive.md (hunk 4 above)
Comment (DanOnCall) [inline articles/quickstart/spa/react/interactive.md]:
```suggestion
Visit the [React Authentication By Example](https://developer.auth0.com/resources/guides/spa/react/basic-authentication/) guide for a deep dive into implementing user authentication in React. This guide provides additional details on how to create a sign-up button, add route guards using React Router, and call a protected API from React.
```

File: articles/quickstart/spa/vuejs/interactive.md [hunk 10]
```diff
@@ -20,6 +20,10 @@ files:
   - files/profile
 ---
 
+:::note
+For a deep dive into implementing user authentication in Vue, visit the [Vue.js Authentication By Example](https://developer.auth0.com/resources/guides/spa/vue/basic-authentication/). This guide provides you with additional details, such as creating a signup button, protecting routes using different strategies, and using class components.
```
Comment (DanOnCall) [inline articles/quickstart/spa/vuejs/interactive.md]:
```suggestion
Visit the [Vue.js Authentication By Example](https://developer.auth0.com/resources/guides/spa/vue/basic-authentication/) guide for a deep dive into implementing user authentication in Vue. This guide provides additional details on how to create a sign-up button, add route guards, and call a protected API from Vue.
```

File: articles/quickstart/spa/react/02-calling-an-api.md [hunk 11]
```diff
@@ -17,6 +17,10 @@ useCase: quickstart
 ---
 <!-- markdownlint-disable MD002 MD034 MD041 -->
 
+:::note
+For a deep dive into making secure calls to an API from React, visit the [Call a Protected API from React](https://developer.auth0.com/resources/guides/spa/react/basic-authentication#call-a-protected-api-from-react/). This guide provides you with additional details, such setting up a sample Express API server and getting test access tokens from the Auth0 Dashboard.
```
Comment (DanOnCall) [inline articles/quickstart/spa/react/02-calling-an-api.md]:
```suggestion
Visit the [Integrate React with an API Server](https://developer.auth0.com/resources/guides/spa/react/basic-authentication#integrate-react-with-an-api-server) section of the [React Authentication By Example](https://developer.auth0.com/resources/guides/spa/react/basic-authentication) guide for a deep dive into calling a protected API from React. This guide allows you to set up a sample API server using a backend technology of your choice, effectively creating a full-stack application.
```

File: articles/quickstart/spa/vuejs/02-calling-an-api.md [hunk 12]
```diff
@@ -18,6 +18,10 @@ useCase: quickstart
 
 <!-- markdownlint-disable MD002 MD041 -->
 
+:::note
+For a deep dive into implementing user authentication in Vue, visit the [Vue.js Authentication By Example: Call an API.](https://developer.auth0.com/resources/guides/spa/vue/basic-authentication#integrate-vue-js-with-an-api-server/). This guide provides you with additional details, such as creating a signup button, protecting routes using different strategies, and using class components.
```
Comment (DanOnCall) [inline articles/quickstart/spa/vuejs/02-calling-an-api.md]:
```suggestion
Visit the ["Integrate Vue.js with an API Server"](https://developer.auth0.com/resources/guides/spa/vue/basic-authentication#integrate-vue-js-with-an-api-server) section of the ["Vue.js Authentication By Example"](https://developer.auth0.com/resources/guides/spa/vue/basic-authentication) guide for a deep dive into calling a protected API from Vue. This guide allows you to set up a sample API server using a backend technology of your choice, effectively creating a full-stack application.
```

File: articles/quickstart/spa/vuejs/01-login.md [hunk 13]
```diff
@@ -20,7 +20,10 @@ useCase: quickstart
 ::: warning
 This quickstart is designed for using [Auth0 Vue](https://github.com/auth0/auth0-vue) with Vue 3 applications. If you are using Vue 2, please check out the [Vue 2 Tutorial with Auth0 SPA SDK](https://github.com/auth0/auth0-vue/blob/main/tutorial/vue2-login.md) instead.
 :::
+:::note
+For a deep dive into implementing user authentication in Vue, visit the [Vue.js Authentication By Example](https://developer.auth0.com/resources/guides/spa/vue/basic-authentication). This guide provides you with additional details, such as creating a signup button, protecting routes using different strategies, and using class components.
```
Comment (DanOnCall) [inline articles/quickstart/spa/vuejs/01-login.md]:
```suggestion
Visit the [Vue.js Authentication By Example](https://developer.auth0.com/resources/guides/spa/vue/basic-authentication) guide for a deep dive into implementing user authentication in Vue. This guide provides additional details on how to create a sign-up button and add route guards to a Vue application.
```

**Review by DanOnCall — CHANGES_REQUESTED** [review]

File: articles/quickstart/spa/react/02-calling-an-api.md (hunk 11 above)
Comment (avanscoy) [reply articles/quickstart/spa/react/02-calling-an-api.md]: Hello! I'm going to push changes in Angular that match this new note ^ @dan-auth0 could you possibly remove the quotation marks from the page/section titles? None of the other entries have this and we honestly don't use quotes around articles in Docs. I couldn't add a suggestion to the batch.

File: articles/quickstart/spa/angular/01-login.md (hunk 6 above)
Comment (avanscoy) [reply articles/quickstart/spa/angular/01-login.md:161]: I added the notes in the new style on the new commit. Sorry about that

File: articles/quickstart/spa/angular/02-calling-an-api.md (hunk 7 above)
Comment (avanscoy) [reply articles/quickstart/spa/angular/02-calling-an-api.md:161]: I also added this one on the new commit.

File: articles/quickstart/spa/react/02-calling-an-api.md (hunk 11 above)
Comment (DanOnCall) [reply articles/quickstart/spa/react/02-calling-an-api.md]: @avanscoy Done 💪

File: articles/quickstart/spa/angular/01-login.md (hunk 6 above)
Comment (DanOnCall) [reply articles/quickstart/spa/angular/01-login.md:161]: Thanks, Amanda. No worries!

File: articles/quickstart/spa/angular/02-calling-an-api.md [hunk 14]
```diff
@@ -18,6 +18,10 @@ useCase: quickstart
 
 <!-- markdownlint-disable MD002 MD034 MD041 -->
 
+:::note
+Visit the [Call a Protected API from Angular](https://developer.auth0.com/resources/guides/spa/angular/basic-authentication#call-a-protected-api-from-angular) section of the [Angular Authentication By Example](https://developer.auth0.com/resources/guides/spa/angular/basic-authentication) guide for a deep dive into calling a protected API from Angular. This guide allows you to set up a sample API server using a backend technology of your choice, effectively creating a full-stack application.
```
Comment (DanOnCall) [inline articles/quickstart/spa/angular/02-calling-an-api.md]:
```suggestion
Visit the [Integrate Angular with an API Server](https://developer.auth0.com/resources/guides/spa/angular/basic-authentication#integrate-angular-with-an-api-server) section of the [Angular Authentication By Example](https://developer.auth0.com/resources/guides/spa/angular/basic-authentication) guide for a deep dive into calling a protected API from Angular. This guide allows you to set up a sample API server using a backend technology of your choice, effectively creating a full-stack application.
```

File: articles/quickstart/spa/vuejs/02-calling-an-api.md [hunk 15]
```diff
@@ -18,6 +18,10 @@ useCase: quickstart
 
 <!-- markdownlint-disable MD002 MD041 -->
 
+:::note
+For a deep dive into making secure calls to an API from Vue, visit the [integrating Vue.js with an API server guide](https://developer.auth0.com/resources/guides/spa/react/basic-authentication#call-a-protected-api-from-react/). This guide provides you with additional details, such setting up a sample Express API server and getting test access tokens from the Auth0 Dashboard.
```
Comment (DanOnCall) [inline articles/quickstart/spa/vuejs/02-calling-an-api.md]:
```suggestion
Visit the [Integrate Vue.js with an API Server](https://developer.auth0.com/resources/guides/spa/vue/basic-authentication#integrate-vue-js-with-an-api-server) section of the [Vue.js Authentication By Example](https://developer.auth0.com/resources/guides/spa/vue/basic-authentication) guide for a deep dive into calling a protected API from Vue. This guide allows you to set up a sample API server using a backend technology of your choice, effectively creating a full-stack application.
```

**Review by DanOnCall — CHANGES_REQUESTED** [review]

File: articles/quickstart/spa/vuejs/01-login.md [hunk 16]
```diff
@@ -20,7 +20,10 @@ useCase: quickstart
 ::: warning
 This quickstart is designed for using [Auth0 Vue](https://github.com/auth0/auth0-vue) with Vue 3 applications. If you are using Vue 2, please check out the [Vue 2 Tutorial with Auth0 SPA SDK](https://github.com/auth0/auth0-vue/blob/main/tutorial/vue2-login.md) instead.
```
Comment (DanOnCall) [inline articles/quickstart/spa/vuejs/01-login.md]:
```suggestion
This quickstart is designed for using [Auth0 Vue](https://github.com/auth0/auth0-vue) with Vue 3 applications. If you are using Vue 2, please check out the [Vue 2 Tutorial with Auth0 SPA SDK](https://github.com/auth0/auth0-vue/blob/main/tutorial/vue2-login.md) instead or visit the [Vue.js Authentication 2 By Example](https://developer.auth0.com/resources/guides/spa/vue/basic-authentication/v2-javascript) guide.
```

**Review by DanOnCall — CHANGES_REQUESTED** [review]

File: articles/quickstart/spa/vuejs/interactive.md [hunk 17]
```diff
@@ -20,6 +20,10 @@ files:
   - files/profile
 ---
 
+:::note
+Visit the [Vue.js Authentication By Example](https://developer.auth0.com/resources/guides/spa/vue/basic-authentication/) guide for a deep dive into implementing user authentication in Vue. This guide provides additional details on how to create a sign-up button, add route guards, and call a protected API from Vue.
+:::
+
```
Comment (DanOnCall) [inline articles/quickstart/spa/vuejs/interactive.md:26]: Could we please update the warning on line `32` as follows:

```
This quickstart is designed for using [Auth0 Vue](https://github.com/auth0/auth0-vue) with Vue 3 applications. If you are using Vue 2, please check out the [Vue 2 Tutorial with Auth0 SPA SDK](https://github.com/auth0/auth0-vue/blob/main/tutorial/vue2-login.md) instead or visit the [Vue.js Authentication 2 By Example](https://developer.auth0.com/resources/guides/spa/vue/basic-authentication/v2-javascript) guide.
```

I am unable to comment or change that line as that is not part of the PR change :)

**Review by DanOnCall — CHANGES_REQUESTED** [review]

File: articles/quickstart/spa/vuejs/interactive.md (hunk 17 above)
Comment (rkhoriander) [reply articles/quickstart/spa/vuejs/interactive.md:26]: Done.

**Review by rkhoriander — DISMISSED** [review]

**Review by frederikprijck — DISMISSED** [review]

File: articles/quickstart/spa/react/01-login.md [hunk 18]
```diff
@@ -140,9 +144,6 @@ export default Profile;
 The `user` property contains sensitive information and artifacts related to the user's identity. As such, its availability depends on the user's authentication status. To prevent any render errors, use the `isAuthenticated` property from `useAuth0()` to check if Auth0 has authenticated the user before React renders any component that consumes the `user` property. Ensure that the SDK has completed loading before accessing the `isAuthenticated` property, by checking that `isLoading` is `false`.
 
 :::panel Checkpoint
-Verify that you can display the `user.name` or [any other `user` property](https://auth0.com/docs/users/references/user-profile-structure#user-profile-attributes) within a component correctly after you have logged in.
+Verify that you can display the `user.name` or [any other `user` property](https://auth0.com/docs/users/references/user-profile-structure#user-profile-attributes/) within a component correctly after you have logged in.
```
Comment (DanOnCall) [inline articles/quickstart/spa/react/01-login.md]:
```suggestion
Verify that you can display the `user.name` or [any other `user` property](https://auth0.com/docs/users/references/user-profile-structure#user-profile-attributes) within a component correctly after you have logged in.
```

**Review by DanOnCall — CHANGES_REQUESTED** [review]

**Review by DanOnCall — APPROVED** [review]
